fwoa-tools/src/migrationUtils.ts

```diff
@@ -166,3 +167,35 @@ export const getFhirClientSMART: () => Promise<AxiosInstance> = async (): Promis
     baseURL
   });
 };
+
+export async function getResource(
+  s3Client: AWS.S3,
+  itemKey: string,
+  bucketName: string
+): Promise<AWS.S3.GetObjectOutput> {
+  console.log(`getting ${itemKey}`);
+  const file = await s3Client
+    .getObject({
+      Bucket: bucketName,
+      Key: itemKey
+    })
+    .promise();
+  if (file.$response.error) {
+    throw new Error(`Failed to get object ${itemKey}`);
+  }
+  return file;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export async function verifyResource(
+  fhirClient: AxiosInstance,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  healthLakeResource: any,
+  resourceId: string,
+  resourceType: string
+): Promise<boolean> {
+  const fwoaResponse = (await fhirClient.get(`/${resourceType}/${resourceId}`)).data;
+  delete fwoaResponse.meta;
+  delete healthLakeResource.meta;
+  return objectHash(fwoaResponse) === objectHash(healthLakeResource);
```

Comment:
Do binary resources have consistency between fwoa and hl?

Reply:
That's a good point. They are consistent except for the data field in the healthlake representation. I can either skip comparing binary resources, or delete the data field and compare in the case of binary

Reply:
It would be better to delete the data field and compare.
Let's also make note of this difference and that it is ignored when verifying in the guide we provide